src/client/windshaft.js

```diff
@@ -72,6 +72,10 @@ export default class Windshaft {
         const MNS = style.getMinimumNeededSchema();
         const resolution = style.getResolution();
         const filtering = windshaftFiltering.getFiltering(style);
+        // FIXME: force to include `cartodb_id` in the MNS columns
```

Comment:
Why Fixme?

Reply:
Just to mark that this is a pseudo-hack and we should revisit it in the future.

Reply:
`FIXME` sounds to me like "FIXME ASAP - possible root of serious bugs and really bad things", not like Maps API requires a little bit of care, but it's your call

Reply:
We can use TODO instead.